src/thermo/WaterPropsIAPWS.cpp

```diff
@@ -144,10 +149,10 @@ doublereal WaterPropsIAPWS::density_const(doublereal pressure,
     if (rhoguess == -1.0) {
         if (phase != -1) {
             if (temperature > T_c) {
-                rhoguess = pressure * M_water / (Rgas * temperature);
+                rhoguess = pressure / (R_water * temperature);
```

Comment:
There are a couple of conditions in the phase diagram that are apparently not be covered by current tests. I have a faint recollection of the model not being usable in some regions?

Reply:
That's correct -- the class's description says it is only meant for use in the liquid region, and I think there are a number of tricky things that would need to be dealt with to extend it beyond that, even though the underlying Helmholtz function is valid everywhere.

Beyond that, improving test coverage for this class was not a goal of this PR. I think the validity of this particular change is clear by inspection.